docker/docker-compose.yml

```diff
@@ -656,6 +694,12 @@ services:
     networks:
       - magistrala-base-net
     volumes:
+      - ./ssl/certs/ca.key:/etc/ssl/certs/ca.key
```

Comment:
Have this as an env variable. Sth like: ` - ./ssl/certs/ca.key:${MG_MQTT_CA_CRT_KEY}`

Reply:
Do the same for the others.